schema.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE item(
+id              INT GENERATED ALWAYS AS IDENTITY,
+publish_date    DATE,
+archived        BOOLEAN DEFAULT FALSE,
+genre_id        INT  REFERENCES genre (id),
+author_id       INT  REFERENCES author(id),
+lable_id        INT  REFERENCES lable(id),
+PRIMARY KEY (id)
+);
+
+CREATE TABLE lable(
+id             INT GENERATED ALWAYS AS IDENTITY,
+title          VARCHAR(100),
+color          VARCHAR(100),
+PRIMARY KEY (id)
+);
+
+CREATE TABLE genre(
+id             INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] Kindly ensure you create INDEX for all your tables because Indexes are special lookup tables that the database search engine can use to speed up data retrieval which comes in handy when you have lots of data 👍🏽